SparkPost/2web2ui
# SA-1497: remove inbox-placement-ea references

### What Changed
 - Removed inbox-placement-ea endpoint replacement

### How To Test
 - enable `use_inbox_placement_ea` ui option. Go to /inbox-placement and go to all of it's pages.
 - verify that the api endpoint is /inbox-placement and not /inbox-placement-ea 

### To Do
- [ ] decommision `use_inbox_placement_ea` feature flag in confluence.
